src/dscigametrics/metric4.py

```diff
@@ -0,0 +1,26 @@
+def find_best_and_worst_campaigns(df, start_date, end_date, campaign_ids, metric):
```

Comment:
please rename your file to your function name "find_best...." and make sure the old file was deleted.